### DISCUSSION TITLE
Multiple Tooltips render for @visx/xychart diagram

### DISCUSSION THREAD

**Discussion by dmytrokruhlyi** [airbnb/visx#1697] [General]
Is it possible to render multiple tooltips for `@visx/xychart` diagram?   
For instance, if I have five `<AnimatedLineSeries />` components I want to display five corresponding tooltips for each line on hover chart.

**Comment by williaster** [comment]
Hi @dimakruhlyi do you have an image of what you want to accomplish? Also this is a usage question so I'm converting to a discussion

**Reply by dmytrokruhlyi** [reply]
@williaster  sure, here is an image. I need to have three tooltips under each line chart. But for now it's possible to see only the nearest one.    

<img width="729" alt="Screenshot 2023-04-26 at 18 28 10" src="https://user-images.githubusercontent.com/32203745/234625106-21243715-b759-49a5-abe2-647b6edd229f.png">

**Comment by nicogreenarry** [comment]
I'm also wondering about this - I'd love to be able to display multiple tooltips. My use case is that I want users to be able to "pin" an arbitrary number of data points, and I want to display persistent tooltips for each pinned data point. Can I specify some points that should have tooltips displayed (until the user unpins them)?

**Reply by nicogreenarry** [reply]
I might be able to use [Annotations](https://airbnb.io/visx/docs/annotation), though, instead of multiple Tooltips.
